libavfilter/vf_vpp_qsv.c

```diff
@@ -518,6 +526,109 @@ static int config_output(AVFilterLink *outlink)
     return 0;
 }
 
+static int update_frame_params(AVFilterContext *ctx, AVFrame *in)
+{
+#if QSV_ONEVPL
```

Comment:
Just for my education purpose, this Marco QSV_ONEVPL is allowed in the code of FFMPEG? Thanks in advance!

Reply:
right.